packages/core/src/components/spinner/_spinner.scss

```diff
@@ -105,6 +105,11 @@ Styleguide components.progress.spinner.js.svg
   }
 }
 
+@keyframes pt-spinner-animation {
+  from { transform: rotate(0deg); }
+  to   { transform: rotate(360deg); }
+}
+
 $spinner-width: $pt-grid-size * 10 !default;
 $spinner-width-factor: 0.5 !default;
 $spinner-width-factor-small: 0.24 !default;
```

Comment:
Slightly off-topic: these values are peculiar to look at. Why were the spinner sizes specified with scale factors instead of simple pixel values?